tests/CMakeLists.txt

```diff
@@ -94,31 +91,38 @@ endfunction()
 
 # Binaries to build
 set(func_binaries
-    "test_map"
-    "test_args_near_unmapped"
-    "test_ddc_overwrite_manager"
-    "test_two_comps"
-    "test_two_comps_inter_call"
+    #"test_map"
+    #"test_args_near_unmapped"
+    #"test_two_comps"
+    #"test_two_comps_inter_call"
     )
 
 set(comp_binaries
     "simple"
-    "time"
-    "lua_simple"
-    "lua_script"
+    #"time"
+    #"lua_simple"
+    #"lua_script"
     "args_simple"
-    "test_ddc_overwrite"
-    "test_two_comps-comp1"
-    "test_two_comps-comp2 0x2000000"
+    #"test_two_comps-comp1"
+    #"test_two_comps-comp2 0x2000000"
     )
 
-# Tests with no arguments
-set(tests_direct
-    "test_map"
-    "test_args_near_unmapped"
-    "test_two_comps"
-    "test_two_comps_inter_call"
-    "test_ddc_overwrite_manager"
+set(tests
+    "simple"
+    #"time"
```

Comment:
Are these intended to be commented out?

Reply:
Can we at least uncomment those that work and for those that don't add a "broken because XYZ" comment at the top?

Reply:
This is the next to do on my list, so if we want that, I can just work on getting them all working and make this a draft temporarily. I was considering delaying raising the PR until I get all tests working anyway.

Reply:
I'd suggest at least getting the easy ones working in this PR as a worthwhile goal.

Reply:
Partly done in https://github.com/capablevms/CHERI-ELF-comp/pull/13/commits/7b6c7899def17930a430699ee3e07877b5b132a6. There is another "easy" test I want to get in, but requires debugging, and that should be all for tests in this PR.

Reply:
I've removed `lua_simple` for now (in https://github.com/capablevms/CHERI-ELF-comp/pull/13/commits/9de13c49ec5f0995fb221688a68142bdf1f48df8). I didn't expect that within a single `so` file, a function call is also done via `plt`, so when we get into lua code, it's trying to runtime load its libraries, and obviously failing. Basically a consequence of #15.